src/excmds.ts

```diff
@@ -1879,7 +1879,15 @@ export async function loadaucmds(cmdType: "DocStart" | "DocLoad" | "DocEnd" | "T
     const aucmds = await config.getAsync("autocmds", cmdType)
     const ausites = Object.keys(aucmds)
     const aukeyarr = ausites.filter(e => window.document.location.href.search(e) >= 0)
+    const owntab = await ownTab()
+    const replacements = {
+        TRI_FIRED_MOZ_TABID: owntab.id,
+        TRI_FIRED_TRI_TABINDEX: owntab.index + 1,
```

Comment:
I've added another variable that gives you the Tridactyl tab index. Perhaps useful with e.g. tabdetach.

It needs documenting before I merge it. I'd like more variables to be added such as the window stuff I mentioned in #2805 too.

I may get round to doing it soon but if you find you have spare time feel free to do it instead : )

Reply:
I've sorted the documentation, I'll have a look at the other variables too. Apart from the window id, any other variables in particular?

Reply:
Perhaps window index so that people can use `taball WININDEX.TABINDEX`? (I'm being lazy with the variable names as I'm on a phone). It's probably not a big use case so don't worry if it's too tricky.

Reply:
You mean tridactyls window index? Because I can't find any information on firefox window index.

Reply:
Yes, Tridactyl's. If you can't work it out really don't worry : )